src/interventions.h

```diff
@@ -48,6 +48,7 @@ void intervention_quarantine_household( model*, individual*, int, int, trace_tok
 void intervention_test_take( model*, individual* );
 void intervention_test_result( model*, individual* );
 void intervention_notify_contacts( model*, individual*, int, trace_token* );
+void intervention_contact_tracing( model*, individual*, int, trace_token* );
```

Comment:
I can't see this function in `interventions.c,` please can your remove.

Reply:
Completed - sorry for the oversight!